### DISCUSSION TITLE
Does suggest ignore the where clause?

### DISCUSSION THREAD

**Discussion by Gomez12** [ravendb/ravendb#16972] [Q&A]
If I run the rql query : 
from "Products"
where LanguageId=1043
select suggest(Product.Title,"nie machine") as title

it creates an auto index which contains the languageid field, but it looks like it is also suggesting terms from other LanguageIds, is this correct behaviour and am I doing something wrong?

Because I would expect this to only suggest items within the LanguageId 1043 constraint.

**Comment by garayx** [comment]
Yes, you query on  `LanguageId` & `Title` fields, so the auto index created will index those fields as well.
You can see more info on Indexes here:
https://ravendb.net/docs/article-page/5.4/csharp/studio/database/indexes/indexes-overview#indexes---the-general-concept
